standard-integration/server.js

```diff
@@ -1,34 +1,149 @@
-import "dotenv/config"; // loads variables from .env file
-import express from "express";
-import * as paypal from "./paypal-api.js";
-const {PORT = 8888} = process.env;
+import express from 'express';
+import fetch from 'node-fetch';
```

Comment:
option to add a `.nvmrc` in this directory with `14` to match [the readme](https://github.com/paypal-examples/docs-examples#have-nodejs-installed)

Reply:
Oh wow Node 14 is no longer even supported. Good call out. I updated the doc to recommend Node 16. We can revisit adding an .nvmrc file and/or validate the minimum required Node.js version on server startup in another PR.